MeasureAuthoringTool/mat-drupal
# Info box and info box grid styles.

## Description
Provide styles for Info Box molecule and Info Box Grid organism. This is for the Latest News & Alerts block shown on the home page mockup.

## To Test
1. Run pattern lab (`npm start` from `html/themes/custom/emat` directory)
2. Visit the Info Box molecule and Info Box Grid molecule and see that the styles match what is shown in the mockups.

<img width="1148" alt="Screen Shot 2020-02-17 at 2 26 50 PM" src="https://user-images.githubusercontent.com/1919263/74684465-9a5b3d80-5191-11ea-8738-00efce541242.png">
